### DISCUSSION TITLE
What's the best way to catch the changes of Appkit(Web3Modal)

### DISCUSSION THREAD

**Discussion by Loganistic** [WalletConnect/walletconnect-monorepo#5227] [Developer Support] [answered]
This is the question about the best way to catch major changes in Appkit(Web3Modal).

I have an experience not recognizing the changes of Web3Modal v3. 
Which is, the format of URI parameter delivered at the deeplink was changed on the Android.

For more detail, it passes URI data on the deeplink by the following.
Web3Modal v2: wc:4f2d8c77a7e06d6924a30a02d...
Web3Modal v3: {wallet's scheme}://wc?uri=wc:4f2d8c77a7e06d6924a30a02d...

But, I couldn't find any notices or documentation of these changes.
What's the effective way to catch the changes of them in the future?

**Comment by glitch-txs** [comment, marked as answer]
hello we won't change this again, we might probably prioritize using Universal Linking over DeepLink in the near future since it's compatible with Telegram Mini Apps

**Reply by Loganistic** [reply]
Okay, thank you for the answer!
